docs/content/en/_index.md

```diff
@@ -4,23 +4,23 @@ date: 2017-03-02T12:00:00-05:00
 features:
   - heading: Blistering Speed
     image_path: /images/icon-fast.svg
-    tagline: What's modern about waiting for your site to build?
+    tagline: What’s modern about waiting for your site to build?
     copy: Hugo is the fastest tool of its kind. At <1 ms per page, the average site builds in less than a second.
 
   - heading: Robust Content Management
     image_path: /images/icon-content-management.svg
-    tagline: Flexibility rules. Hugo is a content strategist's dream.
+    tagline: Flexibility rules. Hugo is a content strategist’s dream.
     copy: Hugo supports unlimited content types, taxonomies, menus, dynamic API-driven content, and more, all without plugins.
 
   - heading: Shortcodes
     image_path: /images/icon-shortcodes.svg
-    tagline: Hugo's shortcodes are Markdown's hidden superpower.
+    tagline: Hugo’s shortcodes are Markdown’s hidden superpower.
     copy: We love the beautiful simplicity of markdown’s syntax, but there are times when we want more flexibility. Hugo shortcodes allow for both beauty and flexibility.
 
   - heading: Built-in Templates
     image_path: /images/icon-built-in-templates.svg
     tagline: Hugo has common patterns to get your work done quickly.
-    copy: Hugo ships with pre-made templates to make quick work of SEO, commenting, analytics and other functions. One line of code, and you're done.
+    copy: Hugo ships with pre-made templates to make quick work of SEO, commenting, analytics and other functions. One line of code, and you’re done.
```

Comment:
I'm not sure whether these different types of apostrophes were used on purpose, but it drove me mad having two types of them next to each other, as in example:

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/5839225/198852095-d14286f8-b05a-4438-ac21-8e0c799d58a3.png">

Please let me know if it makes sense to review the apostrophes across the other docs and/or take a look at some kind of automatised check for this issue.